source/install/install-rhel-71-mattermost.rst

```diff
@@ -0,0 +1,110 @@
+..  _install-rhel-71-mattermost:
+
+Installing Mattermost Server
+============================
+
+Install Mattermost Server on a 64-bit machine.
+
+Assume that the IP address of this server is 10.10.10.2
+
+**To install Mattermost Server on RHEL 7.1**
+
+1. Log into the server that will host Mattermost Server and open a terminal window.
+
+2. Download `the latest version of the Mattermost Server <https://docs.mattermost.com/administration/upgrade.html#version-archive>`_. In the following command, replace ``X.X.X`` with the version that you want to download:
+
+  ``wget https://releases.mattermost.com/X.X.X/mattermost-X.X.X-linux-amd64.tar.gz``
+
+5. Extract the Mattermost Server files.
+
+  ``tar -xvzf *.gz``
+
+3. Move the extracted file to the ``/opt`` directory.
+
+  ``sudo mv mattermost /opt``
+
+4. Create the storage directory for files.
+
+  ``sudo mkdir /opt/mattermost/data``
+
+  .. note::
+    The storage directory will contain all the files and images that your users post to Mattermost, so you need to make sure that the drive is large enough to hold the anticipated number of uploaded files and images.
+
+5. Set up a system user and group called ``mattermost`` that will run this service, and set the ownership and permissions.
+
+  a. ``sudo useradd --system --user-group mattermost``
```

Comment:
During testing, I found that adding my username to the mattermost group didn't allow me to edit the files. Maybe I was doing something wrong, but I couldn't get it to work. The instructions for editing those files say to open as root anyway, so it doesn't make an difference. Also, probably more secure to not add yourself to the group if the only reason is a one-time edit of a file.